backend.native/tests/interop/objc/smoke.h

```diff
@@ -248,4 +248,28 @@ extern BOOL customStringDeallocated;
 @property int value;
 - (int)instanceMethod;
 + (int)classMethod:(int)first :(int)second;
-@end;
\ No newline at end of file
+@end;
+
+// [KT-36067] cinterop tool fails when there is a structure member named Companion
+
+struct EnumFieldMangleStruct {
+	enum {Companion, Any} smth;
```

Comment:
Strict enums (i.e. mapped to Kotlin enums) require testing too.

Reply:
I tested it separately, as well as global variable, but adding all variants into smoke.h would be a syntax error, and adding more separate test cases may be an overkill for that trivial solution. I consider adding mangler tests to `interop/basics` as enums are not objc specific anymore.